.drone.yml

```diff
@@ -3,31 +3,91 @@ kind: pipeline
 name: pipeline1
 
 steps:
+- name: restore-cache
+  image: plugins/s3-cache
+  settings:
+    pull: true
+    access_key:
+      from_secret: drone_aws_access_key_id
+    secret_key:
+      from_secret: drone_aws_secret_access_key
+    restore: true
+    root: TODO-BUCKET-NAME
```

Comment:
@romani Please update all `TODO-BUCKET-NAME` to real S3 bucket name (or reply with bucket name and I will update drone config)